test/integration_test/storagecluster/utils.go

```diff
@@ -0,0 +1,171 @@
+package storagecluster
```

Comment:
Similar to types, we can just a single folder for utils and just use different file names for different kind of utils. It will also help us share some minor util functions.
```
test/
└── integration_test
    └── utils
       ├── storagecluster.go
       ├── fafb.go
       └── common.go
```